### ISSUE TITLE
Cant train profession

### ISSUE THREAD

**Issue by pyroo4ever** [dalaranwow/dalaran-wow#4466]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. i walk up to Therum Deepforge and want to learn blacksmithing and clicks train me, the lowest bs i can train is journeyman blacksmithing. something is wrong, the same is with mining.
2. well i should be able to learn them
3. Profit`?

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by pyroo4ever** [comment]
i just made a new human rogue and went to the closest trainer and i could learn appratice with it but not on my paladin

**Comment by Rushor** [comment]
https://github.com/dalaranwow/dalaran-wow/issues/1483 and https://github.com/dalaranwow/dalaran-wow/issues/2084

Please contact a gamemaster that reviews your current skills. he will investigate it and adds you the spell if needed.

Please write an Ingame-issue to a Gamemaster and the Gamemaster will then help you as soon as he can. Thank you.

![GitHub Logo](http://i.imgur.com/bYkYeGQ.png)
